phase2/addon/lib/timer.js

```diff
@@ -392,21 +385,21 @@ const debug = {
         break;
       case "issilent":
         return isSilent();
-        break;
+        // break;
```

Comment:
The "break" is unreachable, yet I did not want to remove it so that if the code changes later, we wouldn't miss the break. Should I remove it?